.mergify.yml

```diff
@@ -75,13 +79,13 @@ pull_request_rules:
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
   - name: Automatic squash and merge with success checks and the file docker-compose.yml is modified.
     conditions:
-      - check-success=apm-ci/pr-merge
       - label=automation
       - files=docker-compose.yml
+      - check-success=apm-ci/pr-merge
```

Comment:
This change is not required, can you put it back in the above line?

Reply:
moved - check-success=beats-ci/pr-merge to the begin of the list